.cirrus.yml

```diff
@@ -167,15 +167,15 @@ macos_catalina_task:
     - git submodule update --recursive --init
 
   install_dependencies_script:
-    - brew install llvm bison flex cmake ninja python@3.8 sphinx-doc doxygen ccache zeek
+    - brew install llvm bison flex cmake ninja python@3.8 sphinx-doc doxygen ccache
     - pip3 install "btest>=0.66" sphinx_rtd_theme
 
   configure_script:
-    - ./configure --generator=Ninja --with-bison=/usr/local/opt/bison --with-flex=/usr/local/opt/flex --enable-ccache --enable-werror
+    - ./configure --generator=Ninja --with-bison=/usr/local/opt/bison --with-flex=/usr/local/opt/flex --enable-ccache --enable-werror --build-zeek-plugin=no
   build_script:
-    - ninja -C build all build-spicy-analyzers check
+    - ninja -C build all check
   test_build_script:
-    - make -C tests test-all-build
+    - make -C tests test-spicy-build
```

Comment:
`make test` is equivalent to `make test-spicy-build`. I suggest we use `test` as (1) it's more intuitive (in the sense of "don't care what specifically we're testing"), and (2) easy to change what it binds to in the Makefile if we wanted to later.